src/assets/buson.svg

```diff
@@ -0,0 +1,19 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M7.25 12C7.25 11.5858 7.58579 11.25 8 11.25H11C11.4142 11.25 11.75 11.5858 11.75 12C11.75 12.4142 11.4142 12.75 11 12.75H8C7.58579 12.75 7.25 12.4142 7.25 12Z" fill="url(#paint0_linear_1_259)"/>
```

Comment:
The paths reference gradients by id (e.g. url(#paint0_linear_1_259)). If these SVGs are ever inlined into the DOM (imported as React components), duplicated ids will cause collisions when multiple instances are present.

**Sugerencia:**
```typescript
Either keep using them as external files (img src) or ensure ids are unique per-instance (tooling can add hashes) or convert gradients to CSS/less-reusable primitives. If your build imports SVGs as components in other places, prefer unique ids or use a utility to namespace ids on import.
```